Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -1388,10 +1388,10 @@ def build_query_filter(args):
     filtering_args = {
         'minConfidence': args.get('min_confidence', ''),
         'maxConfidence': args.get('max_confidence', ''),
+        'keywords': args.get('keywords', ''),
```

Comment:
please undo moving it (move it below `sources`) - easier to track in git-blame in the future.

Reply:
Done